x-pack/plugins/cases/server/routes/api/cases/similar.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CASES_URL } from '../../../../common/constants';
+import { createCaseError } from '../../../common/error';
+import { createCasesRoute } from '../create_cases_route';
+import type { caseApiV1 } from '../../../../common/types/api';
+
+export const similarCaseRoute = createCasesRoute({
+  method: 'post',
+  path: `${CASES_URL}/_similar`,
+  routerOptions: {
+    access: 'public',
+    summary: `Similar cases`,
+    tags: ['oas-tag:cases'],
+  },
+  handler: async ({ context, request, response }) => {
+    try {
+      const caseContext = await context.cases;
+      const casesClient = await caseContext.getCasesClient();
+
+      const options = request.body as caseApiV1.SimilarCasesSearchRequest;
+
+      const res: caseApiV1.CasesFindResponse = await casesClient.cases.similar({ ...options });
```

Comment:
I think we should have a dedicated response schema for the `similar` route. The `CasesFindResponse` includes 

```
 count_open_cases,
 count_in_progress_cases,
 count_closed_cases,
```

which are not relevant to our case.

What about returning the same schema as `CasesFindResponseRt` but without `CasesStatusResponseRt`?